Sources/MongoSwift/MongoCollection+BulkWrite.swift

```diff
@@ -0,0 +1,131 @@
+import libmongoc
+
+/// An extension of `MongoCollection` encapsulating bulk write operations.
+extension MongoCollection {
+    /**
+     * Sends a batch of writes to the server at the same time.
+     *
+     * - Parameters:
+     *   - requests: an `[WriteModel]` containing the writes to perform
+     *   - options: optional `BulkWriteOptions` to use while executing the operation
+     * - Returns: a `BulkWriteResult`
+     * - Throws: an error if `requests` is empty, or if any error occurs while performing the writes
```

Comment:
Should the error type ``MongoError.typeError` be noted here? I noticed that was done in for `Document.get` and I did the same in ReadPreference docs.